Documentation/manual-olm-install.md

```diff
@@ -0,0 +1,141 @@
+# Manual Installation using Operator Lifecycle Manager (OLM)
+
+Currently Metering via OLM is only supported on Openshift 4.x via the Openshift Marketplace.
+If you want to install metering into a non-Openshift Kubernetes cluster, please use the [manual installation documentation][manual-install].
+
+## Install
+
+This will cover installing the metering-operator via the openshift-marketplace using `kubectl`/`oc` and will then create a Metering resource defining the configuration for the metering-operator to use to install the rest of the Metering stack.
+
+### Install Metering Operator
+
+Installing the metering-operator is done by creating a `Subscription` resource in a namespace with a `CatalogSource` containing the `metering` package.
+
+Currently we advise metering is installed into its own namespace which requires some setup.
+
```

Comment:
Super minor: Currently we advise that ... or Currently we recommend that

Reply:
Fair enough. That said, I didn't change this doc at all, I just renamed it, as I said in the PR summary. It just shows up as a diff.